receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,319 @@
+    enabled: true
+    description: The amount of memory used by the JVM for the direct buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.memory.direct.total_capacity:
+    enabled: true
+    description: The total capacity of all buffers in the direct buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.memory.mapped.used:
+    enabled: true
+    description: The amount of memory used by the JVM for the mapped buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.memory.mapped.total_capacity:
+    enabled: true
+    description: The number of buffers in the mapped buffer pool.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.memory.managed.used:
+    enabled: true
+    description: The amount of managed memory currently used.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.memory.managed.total:
+    enabled: true
+    description: The total amount of managed memory.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.threads.count:
+    enabled: true
+    description: The total number of live threads.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.jvm.garbage_collector.collection.count:
+    enabled: true
+    description: The total number of collections that have occurred.
+    unit: "{collections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: [ garbage_collector_name ]
+  flink.jvm.garbage_collector.collection.time:
+    enabled: true
+    description: The total time spent performing garbage collection.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: [ garbage_collector_name ]
+  flink.jvm.class_loader.classes_loaded:
+    enabled: true
+    description: The total number of classes loaded since the start of the JVM.
+    unit: "{classes}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flink.job.restart.count:
+    enabled: true
+    description: The total number of restarts since this job was submitted, including full restarts and fine-grained restarts.
```

Comment:
This full vs fine-grained be represented as an attribute?

Reply:
I don't believe there are any relevant attributes that can be used. This is flinks description of the NumRestart